static/coffee/directives.coffee

```diff
@@ -59,3 +59,15 @@ directives.directive('cpFieldvalue', () ->
 directives.directive('cpPod', -> {
   templateUrl: -> '/sitestatic/templates/pod.html'
 })
+
+#=====================================================================
+# Tooltip directive
+# Shows 'displayText' with a tooltip at 'position' containing 'tooltipText'
+#=====================================================================
+directives.directive('cpDateTooltip', () ->
+  return {
+    restrict: 'E',
+    scope: {time: '=', position: '@'},
+    templateUrl: '/sitestatic/templates/tooltip.html',
```

Comment:
If you're ok with renaming directive to cp-date then let's also rename template to date.html